### ISSUE TITLE
Dev Test: Magnification features

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#419]
@emily-phet @terracoda @KatieWoe magnification is ready for a QA test.

**[Link to sim](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-customZoom.7/phet/molecules-and-light_en_phet.html?a11y)**

For https://github.com/phetsims/scenery/issues/974

Magnification should behave as you might expect in a normal web application, and input should behave as follows:
 - Keyboard control:
   - 'Ctrl'+ '+'/'-' to zoom in and out (Windows)
   - 'Ctrl' + '0' to sim to initial zoom (Windows)
   - 'Cmd' + '+'/'-' to zoom in and out (macOS)
   - 'Cmd' + '0' to reset zoom (macOS)
   - Arrow keys to pan around screen (Both macOS and Windows)
 - Mouse Control:
   - 'Ctrl' + 'mouse wheel' to zoom in and out (Windows)
   - 'Cmd' + 'mouse wheel' to zoom in and out  (macOS)
   - 'mouse wheel' to pan around screen
- Track pad (Both macOS and Windows, if available):
  - Pinch to zoom in and out
  - Scroll with two fingers on trackpad to pan around screen
-  Touch Screen (Both macOS and Windows)
   - Pinch to zoom in and out
   - Drag with one finger to pan around screen
 - Focus:
   - If focus changes while zoomed in, sim should pan to newly focused item.

Test these methods of zoom control thoroughly and in combination. There will likely be platform differences so please test at least once on each platform combination we support. It would also be good to know how this impacts performance so also test on some of our slowest platforms as well. Finally, please test on an iPhone and Android phone, as we hope use this feature to improve usability on small screens.

If you encounter any issues, create them in github.com/phetsims/molecules-and-light/issues and reference this one.

Thanks! And let me know if you have any questions.

Assigning @ariel-phet to give priority to this issue.

**Comment by ariel-phet** [comment]
@KatieWoe high priority (but after current high priority RC tests of fractions/gas props related sims)

**Comment by KatieWoe** [comment]
Test Matrix:
https://docs.google.com/spreadsheets/d/1I-v15bpG9O9MIMlVhxIPEpJAQ3JHHXbgp5p6CbVPJpE/edit#gid=877256552

**Comment by emily-phet** [comment]
@ariel-phet @KatieWoe 
Can we get an update on estimated timeline for this? It looks like there are two Fractions sims in the queue, will those be tested prior to this?

**Comment by KatieWoe** [comment]
Sorry, the QA repo has been pretty full with a number of high priority things.  I'll start taking a closer look today.

**Comment by KatieWoe** [comment]
Note for:
> Finally, please test on an iPhone and Android phone, as we hope use this feature to improve usability on small screens.

We do not have an Android phone test device.  We have an Android tablet, and an iPhone test device, but that's it.  Is that sufficient for testing @jessegreenberg?

**Comment by jessegreenberg** [comment]
Thanks team, closing this issue.
